client/src/components/sharedComponents/StarRating.jsx

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import roundRating from './lib/getNearestRating.js';
 
 function StarRating({ rating, count, link }) {
+  if (count === 0) return null;
```

Comment:
Shouldn't return rating if no ratings!

Reply:
Is that how it usually is? For some reason I thought that we should display empty stars when there are no ratings as a placeholder, but I'm not entirely sure. Also, in case it's spelled out somewhere in the BRD and I missed it, ignore my comment.